ui/vic-ui-service/src/main/java/com/vmware/vicui/VicUIServiceImpl.java

```diff
@@ -137,8 +136,17 @@ private ServerInfo getServerInfoObject(String serverGuid) {
 		return null;
 	}
 
+    private void setThumbprint(ServerInfo sinfo) {
+        String thumbprint = sinfo.thumbprint;
+        if (thumbprint != null) {
+            _thumbprints.add(thumbprint.replaceAll(":", "").toLowerCase());
+        }
+        ThumbprintTrustManager.setThumbprints(_thumbprints);
```

Comment:
Instead of setting the whole collection, can we not have a function that just adds a new thumbprint to `ThumbprintTrustManager`?